cobra/io/yaml.py

```diff
@@ -4,7 +4,7 @@
 
 import io
 
-from ruamel.yaml import YAML
+from ruamel.yaml.main import YAML
```

Comment:
If you exchange that line with the following one the CI will pass (imports need to be sorted alphabetically).